module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    kmax = 20
</pre><p>It might be better style to use <code>k_max</code>  rather than <code>kmax</code>  because that's the argument passed to freud. Same for <code>kmin</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>